rpc/rpc-wrappers.go

```diff
@@ -1147,7 +1147,7 @@ func NewStorageAuthorityServer(rpc Server, impl core.StorageAuthority) error {
 	})
 
 	rpc.Handle(MethodAddSCTReceipt, func(req []byte) (response []byte, err error) {
-		var sct core.RPCSignedCertificateTimestamp
+		var sct core.SignedCertificateTimestamp
```

Comment:
Does this fit in the deployability guidelines?

I think there was a lot of back and forth on this and I didn't see the final answer.